File: Pratica 6 - Funcoes 2/cpp_logic/exercicio8_v2.cpp
#include <iostream>
using namespace std;

string tipo(float lado_a, float lado_b, float lado_c) {
    if (lado_a == lado_b && lado_b == lado_c) return "Equilátero";
    else if (lado_a == lado_b || lado_a == lado_c || lado_b == lado_c) return "Isósceles";
    else return "Escaleno";
}

int main() {
    float lado_a, lado_b, lado_c;
    cout << "Digite o valor do lado A: ";
    cin >> lado_a;
    cout << "Digite o valor do lado B: ";
    cin >> lado_b;
    cout << "Digite o valor do lado C: ";
    cin >> lado_c;

    cout << tipo(lado_a, lado_b, lado_c);
    return 0;
}

File: Pratica 6 - Funcoes 2/python_logic/exercicio8_v2.py
def classificar_triangulo(lado_a, lado_b, lado_c):
    if lado_a == lado_b == lado_c:
        print("Equilátero.")
    elif lado_a == lado_b or lado_a == lado_c or lado_b == lado_c:
        print("Isósceles.")
    else:
        print("Escaleno.")

lado_a = float(input("Digite o comprimento do lado a: "))
lado_b = float(input("Digite o comprimento do lado b: "))
lado_c = float(input("Digite o comprimento do lado c: "))

classificar_triangulo(lado_a, lado_b, lado_c)